src/bio/bio.pb-c.c

```diff
@@ -0,0 +1,144 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
+/* Generated from: bio.proto */
+
+/* Do not generate deprecated warnings for self */
+#ifndef PROTOBUF_C__NO_DEPRECATED
+#define PROTOBUF_C__NO_DEPRECATED
+#endif
+
+#include "bio.pb-c.h"
+void   srv__bio_error_req__init
+                     (Srv__BioErrorReq         *message)
+{
+  static const Srv__BioErrorReq init_value = SRV__BIO_ERROR_REQ__INIT;
+  *message = init_value;
+}
+size_t srv__bio_error_req__get_packed_size
+                     (const Srv__BioErrorReq *message)
+{
+  assert(message->base.descriptor == &srv__bio_error_req__descriptor);
+  return protobuf_c_message_get_packed_size ((const ProtobufCMessage*)(message));
```

Comment:
(style)  line over 80 characters
(style)  space prohibited between function name and open parenthesis '('